index.js

```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```

Comment:
hmm - I wonder if that means the unsetting could be unnecessary?
It'd be `null` if any ancestor was removed.